### ISSUE TITLE
Plots for bbob-constrained

### ISSUE THREAD

**Issue by brockho** [numbbo/coco#1469]
Similar to the documentation of the `bbob` and `bbob-biobj` test suites, it will be nice to visualize the new `bbob-constrained` problems. Together with @asmaatamna, @anneauger, and @nikohansen, we identified the following:

* Plots of the level sets (in 2-D) of `f` plus the feasible/infeasible domain.
* Potentially plots of `f` value along the constraints: in 2-D as simple plots and in 3-D as `f` level sets on the "constraints equal zero" manifolds (in both cases, we can use the bijective nature of the non-linear constraint transformations to map the non-linear manifolds back to a linear map).
* In addition, we can display the percentage of feasible vs. infeasible points within an `$\epsilon$` ball around the constrained optimum.

**Comment by brockho** [comment]
see also #1665

**Comment by asmaatamna** [comment]
An ```iPython notebook``` for visualizing contours of ```bbob-constrained``` problems in ```2D``` is now available in ```coco-doc``` (see commit #95b30c38eb3063b75ef8e84e36c4c65f3ad3c7d0)

**Comment by brockho** [comment]
I checked your script yesterday and find the resulting plots like what we originally wanted. Thanks.

A few hints how to further improve though (if this is not too much time consuming):
* labels and tick labels have different font sizes on my screen
* the original `bbob` function should be in the title, as well as the number of constraints
* if the f-contour lines could be thinner/lighter in the constrained, the plot will be likely more readable (not sure if we can simply get this by plotting the g-contours on top of the other lines with some transparency).
* The position of the `x_opt` label could be adapted to where the unconstrained region is (with a personal preference to have the label in the constrained region to read better the lines in the unconstrained part).

**Comment by asmaatamna** [comment]
@brockho in the last bullet, do you mean feasible (resp. infeasible) region by constrained (resp. unconstrained)?

**Comment by brockho** [comment]
Yes. The label should be in the infeasible region in order to read better what is going on in the feasible part around the optimum.

**Comment by asmaatamna** [comment]
As suggested by @nikohansen, a good idea would be to look at biobjective plots, which are similar.
